dune_client/api/extensions.py

```diff
@@ -37,7 +38,7 @@
 POLL_FREQUENCY_SECONDS = 1
 
 
-class ExtendedAPI(ExecutionAPI, QueryAPI, TableAPI):
+class ExtendedAPI(ExecutionAPI, QueryAPI, TableAPI, CustomEndpointAPI):
```

Comment:
We may want to log an issue at some point about this multiple inheritance. This design is a pattern that most languages wouldn't allow.

Reply:
Agree. will do internally .